src/autolabel/tasks/attribute_extraction.py

```diff
@@ -235,7 +235,21 @@ def parse_llm_response(
                     error_message=str(e),
                 )
 
-        # TODO(rajas): Handle output guidelines not followed error (for options case)
+        if successfully_labeled:
+            for attribute in self.config.attributes():
+                attr_options = attribute.get("options")
+                if attr_options is not None:
+                    attr_label = llm_label.get(attribute["name"])
+                    if attr_label is not None and attr_label not in attr_options:
+                        logger.warning(
+                            f"Attribute {attr_label} from the LLM response {llm_label} is not in the labels list"
+                        )
+                        successfully_labeled = False
+                        error = LabelingError(
+                            error_type=ErrorType.OUTPUT_GUIDELINES_NOT_FOLLOWED_ERROR,
+                            error_message=f"LLM response is not in the labels list.",
+                        )
+                        llm_label = self.NULL_LABEL_TOKEN
```

Comment:
should we just change this particular subtask to None instead of changing the entire llm_label